AryanGuragain/EventManagement
# V1.2

This pull request includes the addition of the UI.py file, which introduces a user-friendly graphical interface for the Event Management System using Python's Tkinter library. The enhancements include:

    Event Cards: The main window displays events as cards, with key details like event name, date, location, and ticket price. Each card is clickable, leading to a detailed view where users can purchase tickets.
    Responsive Layout: The layout of event cards is dynamic, adjusting the number of columns based on the screen size to ensure a smooth user experience on both large and small screens.
    Ticket Purchase: A dedicated purchase window is provided for users to select the number of tickets and confirm their purchase. Ticket availability and total price are dynamically updated.
    Minimalist Design: A refined, minimalistic color palette has been applied to ensure the interface is professional and visually appealing.
    Error Handling: Improved error handling to manage issues with image loading and event data.

Changes Made:

    Added a new Tkinter-based GUI interface (UI.py) for managing events and ticket purchases.
    Integrated dynamic resizing of event cards based on screen width.
    Updated event display to show images, event details, and ticket prices.
    Implemented ticket purchase confirmation using a dialog window.

How to Test:

    Clone this branch and navigate to the directory where the project is located.
    Run UI.py:

    python UI.py

    The main window will appear with a list of events.
    Click on any event card to open the ticket purchase window.
    Select the number of tickets and confirm the purchase.

Related Issues:

    [Issue #xx] - Issue related to ticket booking interface.
    [Issue #yy] - Issue with event display layout.

Additional Notes:

    Ensure the images for events exist in the specified directory paths for proper display.
    The layout dynamically adjusts based on window size to ensure responsiveness.
